src/workspacer.Watcher/ConsoleHelper.cs

```diff
@@ -27,6 +27,9 @@ public static bool IsConsoleShowing
                 if (handle != IntPtr.Zero)
                 {
                     var style = Win32.GetWindowLongPtr(handle, Win32.GWL_STYLE);
+                    if (Environment.Is64BitProcess) {
+                        return ((UInt64)style & (UInt64)Win32.WS.WS_VISIBLE) != 0;
+                    }
```

Comment:
This fixes the getter code. Do we need to fix the setter code too?

Reply:
I think we need, but I haven't caught any error on that part. 😄

Most awkward part is I don't know how to write C#. I fixed the issue by googling. 😂

Reply:
So... Should I interpret that to mean you won't be able to fix this? And that someone else will have to help out?

CC: @tlsomers - Can you help out here?

Reply:
I see that this functionality has already been implemented in `Win32. GetWindowStyleLongPtr`, why not use that method and the [Enum.HasFlag](https://docs.microsoft.com/en-us/dotnet/api/system.enum.hasflag?view=net-6.0) method.